lib/compat/wordpress-6.0/rest-api.php

```diff
@@ -19,7 +19,7 @@ function gutenberg_register_global_styles_endpoints() {
  * Registers the block pattern directory.
  */
 function gutenberg_register_rest_pattern_directory() {
-	$pattern_directory_controller = new Gutenberg_REST_Pattern_Directory_Controller();
+	$pattern_directory_controller = new Gutenberg_REST_Pattern_Directory_Controller_6_0();
```

Comment:
We wont need this here. We can move this to `6.2/rest-api.php` and init there just the 6_2 controller.